app/persistence/postgreSQL/PgService.ts

```diff
@@ -562,4 +562,30 @@ export class PgService {
 			});
 		});
 	}
+
+	/**
+	 *
+	 *
+	 * @param {*} sql
+	 * @param {*} values
+	 * @returns
+	 * @memberof PgService
+	 */
+	getRowsCountBySQlQuery(sql, values) {
```

Comment:
Please refactor this function to return number only, this way you will always get the number are return type.
```
	async getRowsCountBySQlQuery(sql, values) : Promise<number> {
     const res = await this.client.query(sql, values);
     return res.rowCount;
    }
```

Reply:
modified getRowsCountBySQlQuery to return only the number.